### PR TITLE
Change nextcord.Client to commands.Bot in examples : fix issue #494

### LINKED ISSUE #494: Change: in examples, use commands.Bot instead of nextcord.Client
### Summary

Changing from nextcord.Client to commands.Bot in examples

### What is the feature request for?

The documentation

### The Problem

The client class is not suitable for vast majority of bots, it would be easier for developers to get started with a example if the example uses the more suitable method before hand.

### The Ideal Solution

Change nextcord.Client to commands.Bot and make the required changes.

### The Current Solution

The developer has to change them manually, its not hard however it would be easier if you get that OOTB.

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by MouseMoosz**
closes #494 

## Summary

Change from nextcord.Client to commands.Bot in application commands examples

Yes, it fixes issue #494 

## Checklist

- [ x ] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [ x ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

File: examples/application_commands/autocompleted_command.py [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import nextcord
 from nextcord import Interaction, SlashOption
+from nextcord.ext import commands
 
 TESTING_GUILD_ID = 123456789  # Replace with your testing guild id
 
-client = nextcord.Client()
+bot = commands.Bot(command_prefix='!')
```
Comment (DenverCoder1) [inline examples/application_commands/autocompleted_command.py:7]: In most other examples, the prefix used is `$`

```suggestion
bot = commands.Bot(command_prefix='$')
```

**Review by DenverCoder1 — CHANGES_REQUESTED** [review]

File: examples/application_commands/autocompleted_command.py (hunk 1 above)
Comment (DenverCoder1) [reply examples/application_commands/autocompleted_command.py:7]: When #101 is merged, the prefix can even be left out entirely

File: examples/application_commands/autocompleted_command.py [hunk 2]
```diff
@@ -1,9 +1,10 @@
 import nextcord
```
Comment (DenverCoder1) [inline examples/application_commands/autocompleted_command.py:1]: This is an used import
```suggestion
```

File: examples/application_commands/slash_command.py [hunk 3]
```diff
@@ -1,22 +1,23 @@
 import nextcord
```
Comment (DenverCoder1) [inline examples/application_commands/slash_command.py:1]: unused import
```suggestion
```

**Review by DenverCoder1 — CHANGES_REQUESTED** [review]

**Comment by DenverCoder1** [comment]
Instead of "Fix issue" in the title, it would be better if you could change that to something like "Change nextcord.Client to commands.Bot in examples"

File: examples/application_commands/autocompleted_command.py [hunk 4]
```diff
@@ -1,9 +1,9 @@
-import nextcord
 from nextcord import Interaction, SlashOption
+from nextcord.ext import commands
 
 TESTING_GUILD_ID = 123456789  # Replace with your testing guild id
 
-client = nextcord.Client()
+bot = commands.Bot(command_prefix='$')
```
Comment (DenverCoder1) [inline examples/application_commands/autocompleted_command.py:6]:
```suggestion
bot = commands.Bot(command_prefix="$")
```

File: examples/application_commands/context_menus.py [hunk 5]
```diff
@@ -1,18 +1,19 @@
 import nextcord
+from nextcord.ext import commands
 
 TESTING_GUILD_ID = 123456789  # Replace with your testing guild id
 
-client = nextcord.Client()
+bot = commands.Bot(command_prefix='!')
```
Comment (DenverCoder1) [inline examples/application_commands/context_menus.py:6]:
```suggestion
bot = commands.Bot(command_prefix="$")
```

File: examples/application_commands/slash_command.py [hunk 6]
```diff
@@ -1,22 +1,22 @@
-import nextcord
 from nextcord import Interaction, SlashOption
+from nextcord.ext import commands
 
 TESTING_GUILD_ID = 123456789  # Replace with your testing guild id
 
-client = nextcord.Client()
+bot = commands.Bot(command_prefix='!')
```
Comment (DenverCoder1) [inline examples/application_commands/slash_command.py:6]:
```suggestion
bot = commands.Bot(command_prefix="$")
```

File: examples/application_commands/sub_commands.py [hunk 7]
```diff
@@ -1,11 +1,12 @@
 import nextcord
+from nextcord.ext import commands
 
 TESTING_GUILD_ID = 123456789  # Replace with your testing guild id
 
-client = nextcord.Client()
+bot = commands.Bot(command_prefix='!')
```
Comment (DenverCoder1) [inline examples/application_commands/sub_commands.py:6]:
```suggestion
bot = commands.Bot(command_prefix="$")
```

**Review by DenverCoder1 — CHANGES_REQUESTED** [review]

**Review by DenverCoder1 — APPROVED** [review]
I approve of this change.

In my opinion, all examples should use `commands.Bot` instead of `nextcord.Client` since `commands.Bot` supports many features that every bot should be taking advantage of and it is confusing for beginners to have `commands.Bot` in one place and `nextcord.Client` in another (I have seen many people make the mistake of adding both to their code when really only `commands.Bot` necessary).

After the PR for adding a default command_prefix is merged, I think it might make sense to remove the command_prefix from these examples.

**Review by MaskDuck — APPROVED** [review]
gud job; through you dont need to open a issue thru

**Comment by MouseMoosz** [comment]
@DenverCoder1

Agreed, a newcomer might get confused by the use of two diffrent methods.

> After the PR for adding a default command_prefix is merged, I think it might make sense to remove the command_prefix from these examples.

I tried to do that in the ones which didn't need a prefix like slash command examples, but command_prefix is a required parm for initialization of the bot.

**Comment by MouseMoosz** [comment]
> gud job; through you dont need to open a issue thru

Thank you!
I am new to contributing to other projects and in a guide I saw that you should make issues for pull request first, this was not a big pr so yea it would have made more sense to just directly send a pr.

**Comment by DenverCoder1** [comment]
Opening issues is a good idea, especially since it allows people to discuss the change before the code is actually written. Some repos even requiring PRs to be related to issues, but in others it's fine to directly open a PR.

> I tried to do that in the ones which didn't need a prefix like slash command examples, but command_prefix is a required parm for initialization of the bot.

Right now it is required to specify command_prefix, but #101 is open to change that.

**Comment by MouseMoosz** [comment]
> Opening issues is a good idea, especially since it allows people to discuss the change before the code is actually written. Some repos even requiring PRs to be related to issues, but in others it's fine to directly open a PR.
> 
> > I tried to do that in the ones which didn't need a prefix like slash command examples, but command_prefix is a required parm for initialization of the bot.
> 
> Right now it is required to specify command_prefix, but #101 is open to change that.

Alright!

That would be a nice change.

**Review by Skelmis — APPROVED** [review]
lgtm

**Review by ooliver1 — APPROVED** [review]

File: examples/application_commands/sub_commands.py [hunk 8]
```diff
@@ -1,11 +1,12 @@
 import nextcord
```
Comment (koala9712) [inline examples/application_commands/sub_commands.py:1]: `from nextcord import Interaction` could be used, for consistency across examples

**Pull request merged**